### DISCUSSION TITLE
Create/use a GitHub Action to remove trailing commas?

### DISCUSSION THREAD

**Discussion by obfuscated-loop** [ScoopInstaller/Scoop#5733] [Ideas]
Hi there,

I noticed that quite a lot of PRs revolve around removing trailing commas from the JSON files. I thought it might make sense to create a workflow that uses a GitHub action to remove these. After a brief search, I wasn't able to find one that exists already - but I still think this would be a good addition and would cut-out a lot of PRs where a single char is removed.

EDIT: We could create a [JavaScript action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action) and use an NPM package such as [strip-json-trailing-commas](https://www.npmjs.com/package/strip-json-trailing-commas)

**Comment by rashil2000** [comment]
The actions were recently updated to detect errors such as trailing commas, see https://github.com/ScoopInstaller/GithubActions/pull/39
